crates/oxc_minifier/tests/closure/substitute_alternate_syntax.rs

```diff
@@ -14,14 +16,27 @@ fn fold_return_result() {
 
 #[test]
 fn undefined() {
-    test("var x = undefined", "var x=void 0");
+    test("var x = undefined", "var x");
+    test("let a = 5, b = undefined, c = true", "let a=5,b,c=!0");
+    test("const x = undefined", "const x=void 0");
     test(
         "var undefined = 1;function f() {var undefined=2;var x = undefined;}",
-        "var undefined=1;function f(){var undefined=2,x=undefined}",
+        "var undefined=1;function f(){var undefined=2,x}",
     );
     test("function f(undefined) {}", "function f(undefined){}");
     test("try {} catch(undefined) {}", "try{}catch(undefined){}");
     test("for (undefined in {}) {}", "for(undefined in {}){}");
     test("undefined++", "undefined++");
     test("undefined += undefined;", "undefined+=void 0");
 }
+
+#[test]
+fn console() {
```

Comment:
Is this from google closure's test suite? Otherwise We can move it to `crates/oxc_minifier/tests/oxc` as part of our own test suite.

Reply:
It's not, I'll move it